examples/output-power-management.c

```diff
@@ -72,22 +73,27 @@ static const struct wl_registry_listener registry_listener = {
 static const char usage[] = "usage: output-power-management [options...]\n"
 	"  -h: show this help message\n"
 	"  -e: turn outputs on\n"
-	"  -d: turn outputs off\n";
+	"  -d: turn outputs off\n"
+	"  -m: turn monitor mode on\n";
```

Comment:
"monitor mode" is a little bit confusing for a tool that configures outputs. Maybe we can change the description to "don't exit and monitor changes"?

Reply:
what about "don't exit and continuously print changes" ?

Reply:
Sure, looks good!

Reply:
or '-w': continousely watch for power mgmt mode chnges. Apart from that looks good to me.

Reply:
I made it as @agx suggested (with -`w`)